tests/rocprof-sys-nic-perf.cmake

```
@@ -38,18 +45,20 @@ set(_nic_perf_environment
     "ROCPROFSYS_USE_ROCM=OFF"
     "ROCPROFSYS_TIMEMORY_COMPONENTS=wall_clock,papi_array,network_stats"
     "ROCPROFSYS_NETWORK_INTERFACE=${_network_interface}"
-    "ROCPROFSYS_PAPI_EVENTS=net:::${_network_interface}:tx:byte,net:::${_network_interface}:rx:byte,net:::${_network_interface}:rx:packet,net:::${_network_interface}:tx:packet"
-    )
+    "ROCPROFSYS_PAPI_EVENTS=net:::${_network_interface}:tx:byte net:::${_network_interface}:rx:byte net:::${_network_interface}:rx:packet net:::${_network_interface}:tx:packet"
+    "ROCPROFSYS_SAMPLING_DELAY=0.05")
 
+# Set _download_url to a large file that will give rocprof-sys-sample time to collect NIC
+# performance data.
 set(_download_url
-    "https://github.com/ROCm/rocprofiler-systems/releases/download/rocm-6.3.1/rocprofiler-systems-0.1.0-ubuntu-20.04-ROCm-60200-PAPI-OMPT-Python3.sh"
+    "https://github.com/llvm/llvm-project/releases/download/llvmorg-20.1.0/clang+llvm-20.1.0-armv7a-linux-gnueabihf.tar.gz"
     )
 
 # Run the NIC performance test
 add_test(
     NAME nic-performance
     COMMAND $<TARGET_FILE:rocprofiler-systems-sample> -- wget --no-check-certificate
-            --quiet ${_download_url} -O /tmp/rocprofiler-systems-install.sh
+            --quiet ${_download_url} -O /tmp/rocprofiler-systems.test.bin
```

```suggestion
            ${_download_url} -O /tmp/rocprofiler-systems.test.bin
```
You can remove the `--quiet` flag. This will add runtime to the test, and you won't _need_ to download such a large file.